src/lib/Resources/config/container/solr.yml

```diff
@@ -61,7 +61,10 @@ services:
             - '@ibexa.solr.query.content.facet_builder_visitor.aggregate'
             - '@ibexa.solr.query.content.aggregation_result_extractor.dispatcher'
             - '@Ibexa\Solr\Gateway\EndpointRegistry'
-        deprecated: 'The "%service_id%" service is deprecated since eZ Platform 3.2.0, to be removed in eZ Platform 4.0.0., use ezpublish.search.solr.result_extractor.content.native or ezpublish.search.solr.result_extractor.location.native instead.'
+        deprecated:
+            version: 'eZ Platform 3.2.0'
+            package: 'ibexa/solr'
+            message: 'The "%service_id%" service is deprecated since eZ Platform 3.2.0, to be removed in eZ Platform 4.0.0., use ezpublish.search.solr.result_extractor.content.native or ezpublish.search.solr.result_extractor.location.native instead.'
```

Comment:
If you are touching those, I think we can also change the version (and name) when this gonna be removed.

Reply:
Fixed: https://github.com/ibexa/solr/pull/55/commits/1e89b9e0031f52ccc62354c81ef9089d4800b101